src/MessageScheduler.hpp

```diff
@@ -41,14 +43,14 @@ struct QStringHash {
  */
 class MessageScheduler : public QObject {
   Q_OBJECT
-  bool is_network_unblocked = true;
+  int network_backpressure_counter = 0;
+  const uint64_t max_queue_before_waiting;
```

Comment:
Did this because we can't pull in config.hpp here due to circular imports

Reply:
:thinking: can you remove the MessageScheduler include from RosClientNode? I don't know why that's there and it seems to build without it.